packages/template-retail-react-app/.eslintrc.js

```diff
@@ -6,5 +6,5 @@
  */
 
 module.exports = {
-    extends: require.resolve('pwa-kit-cli/configs/eslint/eslint-config')
+    extends: require.resolve('eslint-config-pwa-kit')
```

Comment:
This is currently failing because lerna is creating a symlink to `eslint-config-pwa-kit/dist`, but the file we want to import is `eslint-config-pwa-kit/index.js`. Moving the file to `eslint-config-pwa-kit/dist/index.js` makes this work as expected. So we need to either (a) commit to dist, which is currently .gitignored, or (2) figure out how to have lerna symlink to the package root instead of `dist`.

Reply:
```suggestion
    extends: ['pwa-kit']
```
I have verified that this `extends` syntax works as expected if `dist/index.js` is created.

(The linting doesn't succeed, because the other node modules can't be found, but the `extends` import works!)